app/controllers/decidim/time_tracker/activities_questionnaire_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Decidim
+  module TimeTracker
+    class ActivitiesQuestionnaireController < ApplicationController
+      include Decidim::Forms::Concerns::HasQuestionnaire
+
+      def answer; end
```

Comment:
Here I'm missing how to handle the answer of the questionnaire. I guess that after saving the answer model, the user has to be redirected to requests_path(activity) (as defined in the time_tracker_controller). I still don't see the correct flow that the has_questionnaire concert follows when answering a questionnaire.